File: examples/node/scrape.mjs
// Scrape a single page with the GhostCrawl Node SDK.
//
//   npm install @ghostcrawl/sdk
//   export GHOSTCRAWL_API_KEY=YOUR_API_KEY
//   node scrape.mjs

import { GhostcrawlClient } from '@ghostcrawl/sdk';

const client = new GhostcrawlClient({ apiKey: process.env.GHOSTCRAWL_API_KEY });

const result = await client.page.scrape({ url: 'https://example.com', engine: 'chrome' });
console.log(result);

File: examples/python/scrape.py
"""Scrape a single page with the GhostCrawl Python SDK.

    pip install ghostcrawl
    export GHOSTCRAWL_API_KEY=YOUR_API_KEY
    python scrape.py
"""

import os

from ghostcrawl import GhostcrawlClient

client = GhostcrawlClient(api_key=os.environ["GHOSTCRAWL_API_KEY"])

result = client.page.scrape(url="https://example.com", engine="chrome")
print(result)
